### PR TITLE
Add store limit guide

### PR DESCRIPTION
**Pull request by rleungx**
### What is changed, added or deleted? (Required)
This PR adds the store limit guide.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: configure-store-limit.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: Store Limit
+summary: Store Limit 使用文档。
+category: how-to
+---
+
+# Store Limit 使用文档
```
Comment (kissmydb) [inline configure-store-limit.md:7]: 这篇文档也放到 SW 中吧，放到教程里。不然切换官网就丢了

**Comment by nolouch** [comment]
需要更新下 TOC ？

File: configure-store-limit.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+title: Store Limit
```
Comment (TomShawn) [inline configure-store-limit.md:2]: 请添加到 TOC 目录中。

File: configure-store-limit.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+summary: Store Limit 使用文档。
```
Comment (TomShawn) [inline configure-store-limit.md:3]:
```suggestion
summary: 介绍 Store Limit 功能。
```

File: configure-store-limit.md (hunk 1 above)
Comment (TomShawn) [inline configure-store-limit.md:7]:
```suggestion
# Store Limit
```

File: configure-store-limit.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+```
+
+示例中，`replace-down-replica` 这个 operator 具体包含了三个操作：
+
+1. 在 store 3 上添加一个 learner peer，id 为 20
+2. 将 store 3 上 id 为 20 的 learner peer 提升为 voter
+3. 删除 store 2 上的 peer
+
+Store Limit 是通过在内存中维护了一个 store ID 到令牌桶的映射，来实现 store 级别的限速。这里对应不同的操作会有不同的令牌桶，目前仅支持限制添加 learner/peer 和删除 peer 两种操作的速度，即对应于每个 store 存在两种类型的令牌桶。
+每次 operator 产生后会检查所包含的操作对应的令牌桶中是否有足够的 token，如果充足才会加入到调度的队列中，同时从令牌桶中拿走对应的 token，否则该 operator 被丢弃。令牌桶会按照固定的速率补充 token，从而实现限速的目的。
```
Comment (TomShawn) [inline configure-store-limit.md:26]:
```suggestion

每次 operator 产生后会检查所包含的操作对应的令牌桶中是否有足够的 token。如果 token 充足才会将该 operator 加入到调度的队列中，同时从令牌桶中拿走对应的 token，否则该 operator 被丢弃。令牌桶会按照固定的速率补充 token，从而实现限速的目的。
```

File: configure-store-limit.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+以上示例中，`replace-down-replica` 这个 operator 具体包含以下操作：
+
+1. 在 `store 3` 上添加一个 learner peer，ID 为 `20`。
+2. 将 `store 3` 上 ID 为 `20` 的 learner peer 提升为 voter。
+3. 删除 `store 2` 上的 peer。
+
+Store Limit 是通过在内存中维护了一个 store ID 到令牌桶的映射，来实现 store 级别的限速。这里不同的操作对应不同的令牌桶，目前仅支持限制添加 learner/peer 和删除 peer 两种操作的速度，即对应于每个 store 存在两种类型的令牌桶。
+
+
```
Comment (TomShawn) [inline configure-store-limit.md:27]:
```suggestion
```

**Pull request merged**